device/thunder_ripple_sdk/src/tests/mock_extension_client.rs

```diff
@@ -0,0 +1,40 @@
+use ripple_sdk::{
+    async_channel::{unbounded, Sender},
+    extn::{
+        client::{extn_client::ExtnClient, extn_sender::ExtnSender},
+        extn_client_message::{ExtnMessage, ExtnPayload, ExtnRequest},
+        extn_id::{ExtnClassId, ExtnId},
+        ffi::ffi_message::CExtnMessage,
+    },
+    framework::ripple_contract::RippleContract,
+    uuid::Uuid,
+};
+
+pub struct MockExtnClient {}
+
+impl MockExtnClient {
```

Comment:
Can we move this to tdk. Actually @bsenth200  is doing this on another effort too. if we can move this to tdk it can be used by both

Reply:
is there an existing place already?